Files/Views/InstanceTabsView.xaml

```diff
@@ -22,12 +22,419 @@
 
     <Grid
         x:Name="rootGrid"
-        Padding="0,8,0,0"
         Background="Transparent">
         <Grid.Resources>
+            <Style x:Key="TabItemStyle" TargetType="muxc:TabViewItem">
```

Comment:
This style should be moved to it's own file in the ResourceDirectories folder in order to keep things organized.